src/main/java/org/jenkinsci/plugins/oic/OicServerManualConfiguration.java

```diff
@@ -262,6 +263,15 @@ public FormValidation doCheckScopes(@QueryParameter String value) {
             return FormValidation.ok();
         }
 
+        @POST
+        public FormValidation doCheckIssuer(@QueryParameter String issuer) {
+            Jenkins.get().checkPermission(Jenkins.ADMINISTER);
+            if (Util.fixEmptyAndTrim(issuer) == null) {
+                return FormValidation.warning(Messages.OicSecurityRealm_IssuerRequired());
+            }
+            return FormValidation.ok();
+        }
+
```

Comment:
not sure why this is here?

Reply:
No idea, I guess some merge problem, just removed (was duplicate)